themes/ddbasic/scripts/menu.js

```diff
@@ -163,15 +163,15 @@
     // If there are too many links in the main menu, so it breaks into
     // two lines, we need to tell CSS to adjust.
     checkMainLinksOffset: function(context) {
-      var main_menu_links = $('.main-menu > li', context);
+      var main_menu_links = $('.main-menu-wrapper > .main-menu > li', context);
```

Comment:
@guddo https://github.com/ding2/ding2/pull/1354

Reply:
Look like they are not done with 4.6.x and they are creating 4.6.2. Therefore we will not release 7.x-4.6.1+easy, instead we will merge 7.x-4.6.2 into development-4.x and create 7.x-4.6.1+easy for release.

When we do that, this change will be merged from upstream - therefore we can remove this in this PR.